index.html

```diff
@@ -58,17 +58,16 @@
           <a href="https://github.com/greenplum-db/gpdb">Code</a>
           <a href="https://github.com/greenplum-db/gpdb-sandbox-tutorials">Sandbox</a>
 
-        </div>
-      </div>
+        </div> </div>
     </div>
 
     <div id='intro-text'>
       <div class='container'>
         <p>
-          The Greenplum Database is an advanced, fully featured, open source data warehouse. It provides powerful and rapid analytics on petabyte scale data volumes. Uniquely geared toward big data analytics, Greenplum Database is powered by the world’s most advanced cost-based query optimizer delivering high analytical query performance on large data volumes.
+          Greenplum Database&reg; is an advanced, fully featured, open source data warehouse. It provides powerful and rapid analytics on petabyte scale data volumes. Uniquely geared toward big data analytics, Greenplum Database is powered by the world’s most advanced cost-based query optimizer delivering high analytical query performance on large data volumes.
         </p>
         <p>
-          The Greenplum project is released under the <a href='http://www.apache.org/licenses/LICENSE-2.0'>Apache 2 license</a>. We want to thank all our current community contributors and are interested in all new potential contributions. For the Greenplum Database community no contribution is too small, we encourage all types of contributions.
+          Greenplum Database&reg; project is released under the <a href='http://www.apache.org/licenses/LICENSE-2.0'>Apache 2 license</a>. We want to thank all our current community contributors and are interested in all new potential contributions. For the Greenplum Database community no contribution is too small, we encourage all types of contributions. To ensure that the use of the Greenplum Database® trademarks and graphics marks will not lead to confusion please follow the <a href="GreenplumDatabaseTrademarkGuidelines.html">Greenplum Database trademark guidelines</a>.
```

Comment:
This is just weird GH rendering of the Pull Request. It looks fine when rendered on the website (just apply the PR and see it in your browser).